lib/CustomPropertyFilters/CustomPropertyFilters.js

```diff
@@ -66,9 +65,9 @@ export default class CustomPropertyFilters extends React.Component {
         closedByDefault
         displayClearButton={numberOfFilters > 0}
         header={FilterAccordionHeader}
-        id="clickable-custprops-filter"
+        id="clickable-customProperties-filter"
```

Comment:
Sorry, but unless we have a really strong reason for it, prefer kebab-casing `id`s.
```suggestion
        id="clickable-custom-properties-filter"
```